gemet/thesaurus/templates/search.html

```diff
@@ -50,6 +49,21 @@
     {% endif %}
   {% endif %}
 
+<div class="pagination">
+    <span class="step-links">
+        {% if concepts.has_previous %}
+            <a href="{% url 'search' language.code concepts.previous_page_number %}?query={{ query }}">previous</a>
+        {% endif %}
+
+        <span class="current">
+            Page {{ concepts.number }} of {{ concepts.paginator.num_pages }}
+        </span>
+
+        {% if concepts.has_next %}
+            <a href="{% url 'search' language.code concepts.next_page_number %}?query={{ query }}">next</a>
+        {% endif %}
+    </span>
+</div>
```

Comment:
One should be able to go to any results page directly. Please use the same layout as the other paginated views use (e.g. Alphabetic Listing).